CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 * None
 
 ### Enhancements
-* None
+* Building for iOS and Android has been optimized for compatibility with future React Native versions, by deferring compilation of JSI dependent code to the dependent app's build.
```

Comment:
- Should we mention that building from source is only possible if you clone the git repository?
- Should we mention that the package now requires a lot more disk space?

Reply:
> Should we mention that building from source is only possible if you clone the git repository?

This isn't different from the current state, so I don't think that's needed here.

> requires a lot more disk space?

Good question 🤔 I'm unsure if that's super relevant as that shouldn't affect the final app's size.

Reply:
Let's leave it as it is